Generator/Templates/GObject/class.sbntxt

```diff
@@ -38,65 +38,58 @@ namespace {{ namespace }}
     
     public partial class {{ name }} {{ write_inheritance }}
     {
-{{if !fundamental}}
+{{~ if !fundamental ~}}
         private static readonly TypeDescriptor GTypeDescriptor = TypeDescriptor.For("{{ type_name }}", {{ name }}.Native.get_type);
 
         #region Signals
         
-        {{- if signals_have_event_args }}
-        
+        {{~ if signals_have_event_args ~}}
         ///<summary>
```

Comment:
There is no space between the triple dash and the `summary` tag.

Reply:
Done